readme.md

```diff
@@ -29,7 +29,7 @@ The plugin streamlines the entire configuration process and removes the need for
 - Jenkins 2.289.1+
 - Bitbucket Server 7.4+
 
-Note: Bitbucket Server 6.10 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+. With 7.0+ you can make use of pull request triggers for jobs. With 7.4+ you can set up an Application Link to have access to all plugin features.
+Note: Bitbucket Server 6.10.17 to 7.3 are also supported, but they're not recommended. This is because some plugin features are not available when using these versions. Instead, we recommend using Bitbucket Server 7.4+. With 7.0+ you can make use of pull request triggers for jobs. With 7.4+ you can set up an Application Link to have access to all plugin features.
```

Comment:
It looks weird to have starting version mentioned upto patch version but the ending version only upto minor version. Please let me know if you have better ideas.

Reply:
Hmm, it is a bit weird, but fine I think. Alternatives I considered.. saying something like `Note: Latest Bitbucket Server 6.10 to 7.3 are also..` but that's not really any better. I think it is fine as is though.